frontend/src/Components/aboutView/AboutViewPage.tsx

```diff
@@ -1,34 +1,33 @@
 import { useEffect, useState } from "react";
 import { connect } from "react-redux";
 import { useParams, Link, useNavigate } from "react-router-dom";
-import { Actions } from "../../Store/Actions/Session.actions";
 import {
   Props as DefaultProperties,
   defaultMapStateToProps,
   defaultMapDispatchToProps,
 } from "../../Interfaces/DefaultConnections";
 import { CardData } from "../../Interfaces/CardData";
+import { Session } from "../../Interfaces/SessionData";
 
 const AboutViewPage = (props: DefaultProperties) => {
-  const [isInit, setInit] = useState(false);
-  const sessionState = props.state.session;
-  const cardArray = props.state.session.cards;
   let ratedCards = 0;
   let checkForRating = false;
   let navigate = useNavigate();
   let {sessionid} = useParams();
+  let isInit = false;
```

Comment:
no longer needed (existence of sessionid is enough), refactor code to not use this